docs/resources/interfaces.md.erb

```diff
@@ -0,0 +1,76 @@
+---
+title: About the interfaces Resource
+platform: os
+---
+
+# interfaces
+
+Use the `interfaces` Chef InSpec audit resource to test the properties of multiple network interfaces on the system.
+
+## Version
+
+This resource first became available in v??? of InSpec.
```

Comment:
@clintoncwolfe Is there available version information for this?

Reply:
@clintoncwolfe Unless told otherwise, I'm going to modify the ??? to have an approximation a la 4.20.x

Reply:
Just removed this entirely per discussion. :)